### DISCUSSION TITLE
Why are there green bars at the bottom using RTSP low, but not high?

### DISCUSSION THREAD

**Discussion by gavsiu** [roleoroleo/yi-hack-MStar#384] [Q&A] [answered]
I have 2 Yi 1080p Home 9FUS. 1 is using version 0.4.3 and the other hasn't been upgraded yet and is on 0.3.4. I also have a Yi 1080p Dome BFUS on 0.3.4.

All 3 cameras have a green bar when I use RTSP low. Is the green bar normal? Any way to get rid of it?

**Comment by roleoroleo** [comment, marked as answer]
It's not normal.
What client are  you using?

**Reply by gavsiu** [reply]
You were right. It was just my client. I was using tinyCAM Pro on my Android tablet and phone. I couldn't test with mpv on my computer, but finally installed VLC to test and it does not exist. Thanks for the tip.
